pycbc/waveform/ringdown.py

```diff
@@ -65,11 +64,60 @@ def props(obj, required, **kwargs):
 
     return input_params
 
+def format_lmns(lmns):
+    """ Checks if the format of the parameter lmns is correct, returning the
+    appropriate format if not, and raise an error if nmodes=0.
+    The required format for the ringdown approximants is a list of lmn modes
+    as strings, with n the number of overtones desired. For instance,
+    lmns = ['223','331'] are the modes 220, 221, 222, and 330.
+    The ConfigParser of a workflow might convert that to a single string
+    (case 1 below) or a list with a single string (case 2), and this function
+    will return the appropriate list of strings. If a different format is
+    given, raise an error.
+    """
+    # Case 0: the lmns is in the right format (a list of strings) or a list
+    # of ints
+    if len(str(lmns[0])) == 3:
+        lmns = [str(lmn) for lmn in lmns]
```

Comment:
@cdcapano This extra step might seem redundant, but it will avoid random python errors if the lmns is a list of ints instead of a list of strings (this could be a user mistake, rather than a workflow re-formatting)

Reply:
@cdcapano with the changes requested today, I ended up removing these lines (it seemed like you didn't want to have them). Let me know if it makes sense to put this back in